### PR TITLE
Updated the ModalForm.jsx

### PR DESCRIPTION
**Pull request by leovaldez08**
Updated the ModalForm by breaking down the steps into two separate components and imported it. Also made some minor changes in Footer.jsx  and README.md

### CODE REVIEW

File: FrontEnd/src/Components/Description.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+          </Text>
+        </Box>
+        <Textarea
+          name="promptDescription"
+          value={formValues.promptDescription}
+          onChange={handleInputChange}
+          color="#333333"
+          bg="white"
+          borderRadius="4px"
+          h="120px"
```
Comment (jithin-kalvian) [inline FrontEnd/src/Components/Description.jsx:95]: For your width, margin, height properties etc., you've used a mix of percentages, pixel values, and the chakra unit-less numbers (which default to pixels). To maintain consistency and allow for easier theme scaling, you might want to stick to one unit or follow a set pattern for when to use which unit.

File: FrontEnd/src/Components/Details.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            mr="30px"
+            _hover={{}}
+          >
+            Back
+          </Button>
+          <Button
+            bg="#6C63FF"
+            color="white"
+            borderRadius="3px"
+            onClick={handleSubmit}
```
Comment (jithin-kalvian) [inline FrontEnd/src/Components/Details.jsx:89]: This component lacks the logic for disabling the Submit button when the form isn't properly filled. Please add it as per the previous page.

File: FrontEnd/src/Components/ModalForm.jsx [hunk 3]
```diff
@@ -24,31 +18,43 @@ function ModalForm({ onClose }) {
...
-      promptType: "",
-      promptName: "",
-      promptDescription: "",
-    });
-    onClose();
-  };
+  const handleSubmit = useCallback(
+    (event) => {
+      event.preventDefault();
+      console.log(formValues);
```
Comment (jithin-kalvian) [inline FrontEnd/src/Components/ModalForm.jsx:44]: Please remove the console logs

**Review by jithin-kalvian — CHANGES_REQUESTED** [review]
Please complete the requested changes

File: FrontEnd/src/Components/ModalForm.jsx (hunk 3 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/ModalForm.jsx:44]: Got it!

File: FrontEnd/src/Components/Details.jsx (hunk 2 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/Details.jsx:89]: Yeah working on it, once I add formik for validation, it'll be resolved!

File: FrontEnd/src/Components/Description.jsx (hunk 1 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/Description.jsx:95]: Noted

File: FrontEnd/src/Components/ModalForm.jsx [hunk 4]
```diff
@@ -90,162 +95,22 @@ function ModalForm({ onClose }) {
...
-                    />
-                  </FormControl>
-                  <Stack direction="row" justifyContent="space-between">
-                    <Button onClick={() => setPage(1)}>Back</Button>
-                    <Button colorScheme="blue" onClick={handleSubmit}>
-                      Submit
-                    </Button>
-                  </Stack>
-                </Box>
+                <Form2
```
Comment (yash0212) [inline FrontEnd/src/Components/ModalForm.jsx:107]: Please use descriptive names for components

File: FrontEnd/src/Components/ModalForm.jsx [hunk 5]
```diff
@@ -90,162 +95,22 @@ function ModalForm({ onClose }) {
...
-                          !formValues.promptName ||
-                          !formValues.promptDescription
-                        }
-                      >
-                        Next
-                      </Button>
-                    </Box>
-                  </Flex>
-                </Box>
+                <Form1
```
Comment (yash0212) [inline FrontEnd/src/Components/ModalForm.jsx:99]: Please use descriptive names for custom components

**Review by yash0212 — CHANGES_REQUESTED** [review]

File: FrontEnd/src/Components/ModalForm.jsx (hunk 4 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/ModalForm.jsx:107]: Got it, will make changes!

File: FrontEnd/src/Components/ModalForm.jsx (hunk 5 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/ModalForm.jsx:99]: Sure

**Pull request merged**